TestFoundation/TestProcess.swift

```diff
@@ -206,7 +207,12 @@ class TestProcess : XCTestCase {
             XCTFail("Could not read stdout")
             return
         }
-        XCTAssertEqual(string, "/bin/cat: invalid_file_name: No such file or directory\n")
+
+        // Remove the leading '/bin/' since on macOS '/bin/cat' just outputs 'cat:'
+        let searchStr = "/bin/"
+        let errMsg = string.replacingOccurrences(of: searchStr, with: "", options: [.literal, .anchored],
+                                              range: searchStr.startIndex..<searchStr.endIndex)
+        XCTAssertEqual(errMsg, "cat: invalid_file_name: No such file or directory\n")
```

Comment:
This smells a bit fragile to me - we're expecting a specific formatting of the error message and text. Why not just assert that the string contains "No such file or directory" instead?

Reply:
Reasonable point, however I would probably wait until it breaks on a specific platform before loosening it. Also, that would also match if the executable didn't exist:

```
$ /bin/foo
-bash: /bin/foo: No such file or directory
```

Reply:
Fair enough :)